### DISCUSSION TITLE
Serving an i386 image to thin client from 64bit system

### DISCUSSION THREAD

**Discussion by mike9208** [ltsp/ltsp#251] [Q&A]
How can I serve multiple images for various thin clients on different architectures.

Server is running ubuntu 19.10
2x NIC

**Comment by alkisg** [comment]
You just create multiple VMs or chroots and you run `ltsp image chrootX` to publish them and `ltsp ipxe` to generate the iPXE menu for them.

Specifically for two images named x86_64 and x86_32, there's also autodetection, 32bit clients will automatically get the x86_32 etc.

**Comment by mike9208** [comment]
Thank you for the quick response, Im doing my first setup, so I guess you can call me a noob..
obviously on install i created a chroot with the x86_64 image what command would I run to add a chroot for x86_32?

**Comment by alkisg** [comment]
You need to read the documentation for all that:

https://ltsp.org/man/ltsp-image/
https://github.com/ltsp/community/wiki/chroots

The easiest way is to use VirtualBox and install for example Ubuntu MATE 18.04 32bit inside it using the .iso image.

**Comment by mike9208** [comment]
Got it, Thanks
